portfolio/src/main/webapp/script.js

```diff
@@ -18,10 +18,10 @@
  */
 const slowFillBiography = async () => {
   const biography_text = 
-      `Hi! My name\'s Thomas, and I\'m just about to start my third year of
-      studies in University College Dublin. I\'m currently a STEP intern
-      at Google, and this portfolio aims to showcase my contributions
-      during my time here.`;
+      'Hi! My name\'s Thomas, and I\'m just about to start my third year of ' +
+      'studies in University College Dublin. I\'m currently a STEP intern ' +
+      'at Google, and this portfolio aims to showcase my contributions ' +
+      'during my time here.';
```

Comment:
The template literal here was removed because of formatting issues. Using a template literal for a multi-line string literal, which is intended to be a single line, will introduce unwanted whitespace.

Reply:
I'd recommend that this goes into a separate pull request, that way you can keep it to 1 change = 1 pull request at a time.

Reply:
I've reverted that commit for this PR now 👍 https://github.com/thomasogara/step-repository/pull/23/commits/267a87afe69155f7cd92ab46e2d5387f85b9388a